snippets/go/list_payments.go

```diff
@@ -49,7 +49,7 @@ func ListPaymentsFiltered(sdk *breez_sdk_liquid.BindingLiquidSdk) {
 func ListPaymentsDetailsAddress(sdk *breez_sdk_liquid.BindingLiquidSdk) {
 	// ANCHOR: list-payments-details-address
 	address := "<Bitcoin address>"
-	details := breez_sdk_liquid.ListPaymentDetailsBitcoin{Address: address}
+	var details breez_sdk_liquid.ListPaymentDetails = breez_sdk_liquid.ListPaymentDetailsBitcoin{Address: address}
```

Comment:
Just to break the line so it can be read on the page better
```suggestion
	var details breez_sdk_liquid.ListPaymentDetails = breez_sdk_liquid.ListPaymentDetailsBitcoin{
		Address: address,
	}
```